### DISCUSSION TITLE
14 febbraio 2023 B - esercizio 2

### DISCUSSION THREAD

**Discussion by IonutCicio** [sapienzastudentsnetwork/basi-di-dati-1#33] [Esami]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/40034429/6283dc14-cf87-4964-af67-0eef824d3a80)

**Comment by Warcophyr** [comment]
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/491ce3dc-a602-4318-a6a8-cb257fe04b01)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/d2a48cf7-5495-4d11-af2f-919266ea73b9)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/f88a1514-1d7d-4181-9584-357cc5b8dd5f)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/6536666b-b3f5-42d6-9951-25b40ecf1b1b)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/8c362782-1707-46d0-aa5b-b63b51b9677c)
![image](https://github.com/sapienzastudentsnetwork/bdi2324/assets/133346488/e79e071e-a1ab-467d-a7d2-e8d9c8849504)
